zorg/buildbot/builders/SphinxDocsBuilder.py

```diff
@@ -176,6 +176,12 @@ def getSphinxRuntimesDocsBuildFactory(
             src_to_build_dir='runtimes',
             **kwargs) # Pass through all the extra arguments.
 
+    f.addStep(ShellCommand(
+                name="Install pip dependencies",
+                command=['pip install -r llvm/docs/requirements.txt'],
```

Comment:
This should be `llvm/docs/requirements-hashed.txt`.

Reply:
0cc13d4ec1c417c5e22312a50975435cb068f13e